app/views/ips/new.html.erb

```diff
@@ -10,6 +10,25 @@
 
     <div class="govuk-grid-column-two-thirds govuk-!-padding-left-0">
       <%= form_with model: @ip do |form| %>
+        <div class="govuk-form-group <%= field_error(@ip, :location_id) %>">
+          <%= form.label :location_id, "Select your location", class: "govuk-label"  %><br />
+          <%= form.select :location_id, @locations, { prompt: '-- select --' }, { class: "govuk-select" } %>
+        </div>
+
+        <p class="govuk-body"><strong>OR</strong></p>
+
+        <div class="govuk-form-group <%= field_error(@ip, :location_address) %>">
+          <%= form.label :location_address, "Enter new location address", class: "govuk-label"  %><br />
```

Comment:
This needs to be `f.fields_for :location` in order to pass the correct params.

See https://github.com/alphagov/govwifi-admin/blob/master/app/views/users/confirmations/show.html.erb#L12